### ISSUE TITLE
mobile issues

### ISSUE THREAD

**Issue by howardchung** [odota/ui#81]
- [ ] navbar runs off the screen instead of stacking
- [ ] home opening text runs offscreen
- [x] player header elements overflow vertically
- [x] TabBar navigation squishes together horizontally instead of wrapping
- [x] overview match/hero tables are squished together (they should stack if the viewport is too narrow?  bootstrap handles this automatically)
- [x] home three features section should stack instead of compressing into 1/3 width
- [ ] Tables are way too wide in mobile scroll view.

**Comment by Roshanjossey** [comment]
![image](https://cloud.githubusercontent.com/assets/8488446/19073566/44c2c26a-8a59-11e6-8a4a-c26abab37079.png)

@mike-robertson: I've annotated the issues mentioned in the screenshot I added above.
Could you please check if I've got them right? Also have we decided on what to do about
>overview match/hero tables are squished together (they should stack if the viewport is too narrow? bootstrap handles this automatically)

or is that an open question?

**Comment by mike-robertson** [comment]
Let's pull @blukai into this discussion. My thinking is 
Navbar: add in hamburger menu for mobile
Player Header: shrink picture and stack mmr/ match info under player name (that can't fit next to name) 
Tables: auto width and stack, scrolling x
Tabbar: allow to run offscreen and swipe/scroll

**Comment by Roshanjossey** [comment]
Cool @blukai: I'll get started with Tabbar. Also when I run
`npm run lint`, I get
>`> opendota-ui@1.0.0 lint /Users/roshan/open/ui`
>`> eslint . --ext .js,.jsx --fix`

>`The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint->plugin-import instead.`

With no errors. Is this how it's supposed to be?

**Comment by blukai** [comment]
@Roshanjossey, eslint just checks the js,  jsx errors by a config, for example if u **skip** semicolon on second line
```js
function mapStateToProps(data) {
  return { content: data.content };
}
``` 
these will be an error.

And I guess navbar items can be wrapped in dropdown.

**Comment by howardchung** [comment]
added a couple of issues with homepage on mobile

**Comment by howardchung** [comment]
replacing with individual issues:
#295 
#278 

The second is fixed by PR #252
